src/nvim/redraw.h

```diff
@@ -0,0 +1,12 @@
+#ifndef NVIM_REDRAW_H
+#define NVIM_REDRAW_H
+
+#include "nvim/api/private/defs.h"
+
+typedef Object (*ApiRedrawWrapper)(Array args);                                     
```

Comment:
In ui_events_redraw.generated.h the functions have return type `void` instead of `Object`. So should `Object` be replaced with `void` ?

Reply:
Yes . It should be `void`.